jsonsaver/saver2v2/save_package.go

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+package saver2v2
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/spdx/tools-golang/spdx"
+)
+
+func renderPackage2_2(doc *spdx.Document2_2, buf *bytes.Buffer) error {
+
+	var packages []interface{}
+	for k, v := range doc.Packages {
+		pkg := make(map[string]interface{})
+		pkg["SPDXID"] = spdx.RenderElementID(k)
+		ann, _ := renderAnnotations2_2(doc.Annotations, spdx.MakeDocElementID("", string(v.PackageSPDXIdentifier)))
+		if ann != nil {
+			pkg["annotations"] = ann
+		}
+		if v.PackageAttributionTexts != nil {
+			pkg["attributionTexts"] = v.PackageAttributionTexts
+		}
+		// parse package checksums
```

Comment:
here and below, there are some references to "parsing" -- looks like these might have been accidentally copied over from the parser? I would probably take those comments out, since this is saving rather than parsing.